capstone-frontend/src/components/Logout.jsx

```diff
@@ -22,24 +20,12 @@ export class Logout extends Component {
 
   render() {
     return (
-      <div id="logout">
-
-        <DropdownButton as={ButtonGroup} title={
-          <img id="profile_img"
-            className="img-responsive rounded-circle"
-            src={this.state.profileObj.imageUrl}
-            alt={this.state.profileObj.name} />}
-
-          id="bg-vertical-dropdown-1">
-
-          <Dropdown.Item eventKey="1">
-            <GoogleLogout
-              clientId="962122785123-t0pm10o610q77epuh9d1jjs29hamm1nf.apps.googleusercontent.com"
-              buttonText="Logout"
-              onLogoutSuccess={this.logoutResponseSuccess}
-              isSignedIn={false} />
-          </Dropdown.Item>
-        </DropdownButton>
+      <div style={this.props.loginStyle || {}}>
```

Comment:
In what case would `<Logout>` not be passed a loginStyle?

Reply:
Well I may not want to apply a style. I think my LeftSideBar doesn't actually apply a style when it uses ```<Logout/>```. I just don't want to limit our self on styling.